aiohttp/connector.py

```diff
@@ -1345,7 +1401,7 @@ async def _create_proxy_connection(
                 if not runtime_has_start_tls:
                     # HTTP proxy with support for upgrade to HTTPS
                     sslcontext = self._get_ssl_context(req)
-                    return await self._wrap_create_connection(
+                    return await self._wrap_existing_connection(
```

Comment:
This path doesn't exist in 4.0 so I renamed the function to keep the code exactly as-is.